index.js

```diff
@@ -20,8 +20,52 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    Recipe.create({title: "Asian Glazed Chicken Thighs",
```

Comment:
Just for good practices, remember to indent the objects and put all properties new lines (If they are multiline objects). Just for having a more clear code. You can also create the object before the promise inside a variable and then just pass that variable inside the promise.